zipkin-server/src/main/resources/zipkin-server.yml

```diff
@@ -29,6 +29,9 @@ zipkin:
     lookback: ${QUERY_LOOKBACK:86400000}
     # The Cache-Control max-age (seconds) for /api/v1/services and /api/v1/spans
     names-max-age: 300
+    # CORS allowed-origins.
+    allowed-origins: "*"
```

Comment:
¯\_(ツ)_/¯ if it's possible for somebody to "suck at yaml", that would be me. Thinking about it a little more, this entry in the config file is a little redundant now since the default value is * now - wasn't sure if it was perhaps better "documentation" to leave it in.